app/models/article.rb

```diff
@@ -0,0 +1,2 @@
+class Article < ActiveRecord::Base
+end
```

Comment:
you will need to add at least one validation in order to test the 'unhappy path' => creating an article without a title or a body.  You can choose to require one or the other, or both.  Alternatively; opt not to do this, opt not to use flash messages, and remove the confirmation message 'successfully created' and simply ensure that the supplied title and body appear in the show page.